### PR TITLE
Fixed copying shadow pages with urls

### LINKED ISSUE #1787: Missing url for copied shadow pages
I have a page in DE, EN and ES. DE and EN have normal content and in ES I set a shadow on EN. The url property of the node is not correct in ES.

### PR DESCRIPTION
**Pull request by danrot**
__tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1787
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
@wachterjohannes please also merge into develop

File: src/Sulu/Component/Content/Mapper/ContentMapper.php [hunk 1]
```diff
@@ -758,14 +758,9 @@ private function copyOrMove($uuid, $destParentUuid, $userId, $webspaceKey, $loca
             $this->documentManager->find($document->getUuid(), $locale);
             $this->documentManager->find($parentDocument->getUuid(), $locale);
 
-            $parentResourceLocator = '/';
-            if ($parentDocument instanceof ResourceSegmentBehavior) {
-                $parentResourceLocator = $parentDocument->getResourceSegment();
-            }
```
Comment (danrot) [inline src/Sulu/Component/Content/Mapper/ContentMapper.php]: This variable have not been used.

**Comment by danrot** [comment]
@c00n84 Can you test this solution?

**Comment by wachterjohannes** [comment]
@danrot styleci fails

**Comment by danrot** [comment]
@wachterjohannes Because it's targeted at master :wink:

**Comment by wachterjohannes** [comment]
@danrot ok (: see it!

**Comment by danrot** [comment]
We won't release a hotfix for that. Already merged in develop.

**Pull request closed without merging**